### ISSUE TITLE
netconf call home

### ISSUE THREAD

**Issue by tcnyambira** [Melacon/ntsim#3]
Apart from enabling each simulated device to use the Call Home feature to connect to the ODL instance, once it boots up, can you also use the NETCONF call home feature for every netconf end point to continuously send device configuration/status data to the controller so that you always have real time or near real time data at the controller

**Comment by alexandrustancu** [comment]
I don't think this is the purpose of the Call Home feature. After the NETCONF connection is established, it is the responsibility of the ODL instance to query the data.
